packages/terra-button/tests/wdio/button-spec.js

```diff
@@ -238,4 +238,18 @@ Terra.describeViewports('Button', ['tiny'], () => {
 
     Terra.it.validatesElement({ selector: '#compactButton' });
   });
+
+  describe('Programmatic Focus', () => {
+    before(() => {
+      browser.url('/#/raw/tests/terra-button/button/programmatic-focus');
+      browser.click('#button1');
+    });
+
+    Terra.it.matchesScreenshot('Button 2 focused programmatically');
```

Comment:
Shouldn't this be button 1?

Reply:
button 1's callback focuses button 2 and vice versa. Button 1 was just clicked so button 2 is now focused.